curp/src/rpc/mod.rs

```diff
@@ -507,7 +515,17 @@ impl ProposeConfChangeRequest {
     #[inline]
     #[must_use]
     pub fn new(id: String, changes: Vec<ConfChange>) -> Self {
-        Self { id, changes }
+        Self {
+            id,
+            changes,
+            cluster_version: 0,
```

Comment:
please add `cluster_version` into the constructor parameter

Reply:
Fixed